pinot-spi/src/main/java/org/apache/pinot/spi/utils/builder/ControllerRequestURLBuilder.java

```diff
@@ -231,6 +231,10 @@ public String forTableForceCommit(String tableName) {
     return StringUtil.join("/", _baseUrl, "tables", tableName, "forceCommit");
   }
 
+  public String forTableToBrokerMappings() {
```

Comment:
Is this related?

Reply:
Oh no, I committed thinking it was some other branch, will remove